Makefile

```diff
@@ -119,18 +119,25 @@ docs/papers-in.html: docs/papers.xml docs/bib2html.xsl
 
 docs/papers.html: docs/papers-in.html
 
-%.html: %.tt layout.tt common.tt nix-release.tt nixos-release.tt donation.tt
+%.html: %.tt layout.tt common.tt nix-release.tt nixos-release.tt donation.tt nix/install-hash.tt
 	tpage \
 	  --pre_chomp --post_chomp \
 	  --define modifiedAt="`git log -1 --pretty='%ai' $<`" \
 	  --define modifiedBy="`git log -1 --pretty='%an' $<`" \
 	  --define root=`echo $@ | sed -e 's|[^/]||g' -e 's|/|../|g'` \
 	  --define fileName=$< \
-	  --pre_process=nix-release.tt --pre_process=nixos-release.tt --pre_process=common.tt \
+	  --pre_process=nix-release.tt --pre_process=nix/install-hash.tt --pre_process=nixos-release.tt --pre_process=common.tt \
 	  $< > $@.tmp
 	xmllint --nonet --noout $@.tmp
 	mv $@.tmp $@
 
+nix/install-hash.tt: nix/install
+	echo "[%-" > $@
+	echo " nixInstallHashsha256 = \"$$(sha256sum nix/install | cut -d' ' -f1)\";" >> $@
+	echo " nixInstallHashsha1 = \"$$(sha1sum nix/install | cut -d' ' -f1)\";" >> $@
+	echo " nixInstallHashmd5 = \"$$(md5sum nix/install | cut -d' ' -f1)\";" >> $@
```

Comment:
I only actually used the sha256sum, the other two were remnants from an experiment. Cleaned up.